inference-engine/src/offline_transformations/include/mask_attribute.hpp

```diff
@@ -28,6 +27,11 @@ namespace ngraph {
 class Mask : public std::vector<std::set<uint64_t>>,
              public std::enable_shared_from_this<Mask> {
 public:
+    static const ::ov::DiscreteTypeInfo& get_type_info_static() {
```

Comment:
Can we use OPENVINO_RTTI?

Reply:
if I am not mistaken `OPENVINO_RTTI` can be used if have already  `virtual get_type_info()` in the class hierarchy. This class does not inherit something.